internal/btf/types.go

```diff
@@ -90,8 +90,9 @@ type Struct struct {
 	TypeID
 	Name
 	// The size of the struct including padding, in bytes
-	Size    uint32
-	Members []Member
+	Size     uint32
+	Members  []Member
+	KindFlag bool
```

Comment:
I think we shouldn't expose KindFlag to users of the package, it's a detail of the BTF encoding. Can you perform the normalization for this at inflation stage?

Basically, `Member` would gain `BitfieldSize` and `BitOffset`.

Reply:
Updated.